front/components/cart/cart.scss

```diff
@@ -0,0 +1,8 @@
+.cart{
+
+  img.picture{
```

Comment:
if there is going to be only a single img tag in the partial, you can remove the class name and just use img